### PR TITLE
Fix/update basisprofil dependency

### PR DESCRIPTION
**Pull request by patrick-werner**
## Description
Basisprofile updated to allign with vitalsigns international.
(Fix for vs-1 invariant, allow effectivePeriod)

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: Resources/input/fsh/ISiK-Koerpertemperatur.fsh [hunk 1]
```diff
@@ -16,14 +19,21 @@ Id: ISiKKoerpertemperatur
 * valueQuantity.system MS
 * valueQuantity.code MS
 
+ValueSet: ISiKKoerpertemperaturSCTVS
```
Comment (f-peverali) [inline Resources/input/fsh/ISiK-Koerpertemperatur.fsh:22]: question: warum bauen wir ein VS mit 1 Value? 
Kann man dann nicht gleich fixed pattern setzen?

File: Resources/input/fsh/ISiK-Kopfumfang.fsh [hunk 2]
```diff
@@ -16,14 +19,21 @@ Id: ISiKKopfumfang
 * valueQuantity.system MS
 * valueQuantity.code MS
 
+ValueSet: ISiKKopfumfangSCTVS
```
Comment (f-peverali) [inline Resources/input/fsh/ISiK-Kopfumfang.fsh:22]: question: s.o.

**Review by f-peverali — CHANGES_REQUESTED** [review]
suggestion: bitte lines zu releasenotes hinzu

Fragen zu VS wären nice (non-blocking)

File: ImplementationGuide/markdown/ReleaseNotes.md [hunk 3]
```diff
@@ -4,6 +4,13 @@ Im Rahmen der ISiK-Veröffentlichungen wird das [Semantic Versioning](https://se
 
 Die erste Ziffer X bezeichnet ein Major-Release und regelt die Gültigkeit von Releases. Die dritte Ziffer Y (Release x.0.y) bezeichnet eine technische Korrektur und versioniert kleinere Änderungen (Packages) während eines Jahres, z. B. 1.0.1.
 
+----
+Version: 4.0.0
```
Comment (f-peverali) [inline ImplementationGuide/markdown/ReleaseNotes.md]:
```suggestion
Version: 4.0.0-rc
```

**Review by f-peverali — APPROVED** [review]

File: Resources/input/fsh/ISiK-Koerpertemperatur.fsh (hunk 1 above)
Comment (patrick-werner) [reply Resources/input/fsh/ISiK-Koerpertemperatur.fsh:22]: Um überall einheitlich ein VS binding einzusetzen

**Pull request merged**